js/editor.js

```diff
@@ -1,6 +1,9 @@
 const { h, render, Component } = preact;
 const { route, Router, Link } = preactRouter;
 
+const REST_URL_temp = 'https://seed-trial.firebaseio.com/user.json';
+const REST_URL = 'https://emrg-pcg.firebaseio.com/';
```

Comment:
I suggest making a `BASE_REST_URL` and a `SKETCH_REST_URL` that appends `/sketch`. This makes it easier in other contexts where we pass a function that loads the sketch ID from somewhere else (e.g. file system).